spec/requests/private_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PrivateController, type: :controller do
+  describe 'GET#index' do
+    context 'when user is not sign in' do
+      it 'returns redirect status' do
+        get :index
+
+        expect(response).to have_http_status(:found)
+      end
```

Comment:
```suggestion
      it 'redirects to the new user session page' do
        get :index

        expect(response).to have_http_status(:found)
        expect(response).to redirect_to(new_user_session_path)
      end
```

(the `new_user_session_path` might be different in your project, check at `rails routes` CLI to see what path is expected) :)